apps/game_client/assets/js/hooks/board_game.js

```diff
@@ -102,6 +103,11 @@ export const BoardGame = function () {
         _this.pushEvent("attack", "3");
         _this.updateDebug("key: " + key);
       }
+
+      if (event.key === "l") {
+        _this.pushEvent("use_item", "1");
+        _this.updateDebug("key: " + key);
+      }
```

Comment:
This is not working and even makes the client crash (or something like it), so any help is welcomed

Reply:
I think this has something to do to the fact that we're not sending a direction when we try to send a skill and the server expects us to when we do `skill_params.target` in the `player:140` module right?

Reply:
No, because this is its own separate thing. It doesn't go through the regular `attack` path

Also, the crash is in the web client, like either browser crashing or something in liveview

Reply:
FYI since the client is crashing already, don't see any issue with merging with this issue and fixing later

Reply:
yeah that was due to the problem i mentioned 😅 but it's a different one. That problem was solved on #312

Reply:
Finally fixed in 3d8dd09